README.md

```diff
@@ -0,0 +1,212 @@
+   SPRING_LOCAL_PORT=8080
+   SPRING_DOCKER_PORT=8080
+   DEBUG_PORT=5005
+   
+   JWT_SECRET=JustAnotherSuperSecretString1234!
+   JWT_EXPIRATION=3600000
+   ```
+4. Install dependencies and build the project:
+
+   ```bash
+   mvn clean install
+   ```
+5. Build and Run the Docker Containers:
+
+   ```bash
+   docker-compose up
+   ```
+6. Access the Application:
+
+   Open your browser and go to `http://localhost:8080/api/swagger-ui/index.html#/`
+   to access the Swagger API documentation.
+7. Stop and Remove Containers:
+
+   To stop and remove the containers created by the Compose file, use the
+   docker-compose down command:
+
+    ```bash
+    docker-compose down
+    ```
+
+## Swagger Documentation
+
+Explore the detailed API documentation generated by Swagger to understand the various endpoints
+and their functionalities.
+
+<details>
+<summary><h3>How to Use Swagger for API Testing</h3></summary>
+  <p>Follow the next steps:</p>
+  <ul>
+    <li>
+         Open Auth Management registration endpoint:
+         <img src="/images/swagger/auth_manager.png" alt="auth manager">
+    </li>
+    <li>
+         Click on the "Try it out" button:
+         <img src="/images/swagger/try_it_out.png" alt="auth manager">
+    </li>
+    <li>
+         Register a new user with unique email and press "Execute"
+         <img src="/images/swagger/register_execute.png" alt="auth manager">
+    </li>
+    <li>
+         If registration passes successfully, you'll see your registration data:
+         <img src="/images/swagger/register_response.png" alt="auth manager">
+    </li>
+    <li>
+         Open Auth Management login endpoint:
+         <img src="/images/swagger/auth_manager_login.png" alt="auth manager">
+    </li>
+    <li>
+         Click on the "Try it out" button:
+         <img src="/images/swagger/try_it_out_login.png" alt="auth manager">
+    </li>
+    <li>
+         Use registered login and password:
+         <img src="/images/swagger/login_execute.png" alt="auth manager">
+    </li>
+    <li>
+         Copy token from response body:
+         <img src="/images/swagger/jwt_token.png" alt="auth manager">
+    </li>
+    <li>
+         Click on the "Authorize" button on the top of the page:
+         <img src="/images/swagger/authorize_btn.png" alt="auth manager">
+    </li>
+    <li>
+         Paste the token in the field, and click "Authorize":
+         <img src="/images/swagger/swagger_auth.png" alt="auth manager">
+    </li>
+    <li>Now you can use all endpoints</li>
+  </ul>
+</details>
+
+### 🔐 Authentication Endpoints:
+
+* Register a new user: `POST /api/auth/register`
+   ```json
+   {
+     "email": "user@mail.com",
+     "password": "password",
+     "repeatPassword": "password",
+     "firstName": "Owly",
+     "lastName": "User",
+     "shippingAddress": "Main St. 57, Lviv"
+   }
+  ```
+* Authenticate a user: `POST /api/auth/login`
+  ```json
+   {
+     "email": "user@mail.com",
+     "password": "password"
+   }
+  ```
```

Comment:
Tell more about the token and how to use it, also add simple user and admin credentials to readme to provide the visitor of your project opportunity to test all endpoints